.travis.yml

```diff
@@ -30,10 +30,10 @@ jobs:
       env: PURPOSE='Dependency Check'
       script: ./scripts/dependency/check.sh
       python: 3.6
-#    - stage: verify
-#      env: PURPOSE='Lint Command Table and Help'
-#      script: ./scripts/ci/verify_linter.sh
-#      python: 3.6
```

Comment:
I would leave this disabled for now and just merge the relocation of the YML files

Reply:
Works for me!

Reply:
Fixed in f07fb88